tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -359,3 +368,56 @@ def fencil(out: Field[[Edge], float64], inp: Field[[Edge], float64], factor: flo
 
     ref = np.arange(1, size + 1) * factor
     assert np.allclose(ref, out.array())
+
+
+def test_broadcast_simple():
+    size = 10
+    a = np_as_located_field(IDim)(np.arange(0, size, 1, dtype=int))
+    out = np_as_located_field(IDim, JDim)(np.zeros((size, size)))
+
+    @field_operator(backend="roundtrip")
+    def simple_broadcast(inp: Field[[IDim], float64]) -> Field[[IDim, JDim], float64]:
+        return broadcast(inp, (IDim, JDim))
```

Comment:
I used a tuple since it is already supported and immutable, at the time @tehrengruber also supported this approach, Till are there any additional reasons to not support definining the broadcast dimensions within a list?

Reply:
I suggested to use a tuple for two reasons: First we do not support parsing lists and I think we shouldn't as the only thing that differentiates them from tuples is their mutability. The other issue is that we retrieve the actual dimensions using the type system. For example the type of `(IDim, JDim)` is `ct.TupleType(types=[ct.DimensionType(dim=Dimension("IDim")), ct.DimensionType(dim=Dimension("JDim"))])` and we get the dimensions from the latter instead of relying on the specific form of the AST. Making use of the type system has the advantage that it is easy to support case like:
```
some_dim = Dimension("K")
broadcast(some_field, (some_dim,))
```
This is a manufactured example, but as soon as one starts to write generic code this can be quite handy. For lists however this does not work easily as the type of a list stores only the common type of all of it's elements, e.g. `List[Dimension]`. The disadvantage of the tuple approach is that it doesn't match how we annotate fields (as you mentioned). Maybe it makes sense to switch to `Field[(...), float64]` at some point.